test/domain/secureMessage/RequestSpec.scala

```diff
@@ -164,13 +164,13 @@ trait Setup {
   val PostponedVATTemplate = "customs_financials_requested_notification_adjustment_statements_not_found"
 
   val TestContents = {
-    List(secureMessage.Content("en", AccountType("DutyDefermentStatement"), DutyDefermentBody("Company Name")),
-      secureMessage.Content("cy", AccountType("DutyDefermentStatement"), DutyDefermentBody("Company Name")))}
+    List(secureMessage.Content("en", "DutyDefermentStatement", DutyDefermentBody("Company Name")),
+      secureMessage.Content("cy", "DutyDefermentStatement", DutyDefermentBody("Company Name")))}
 
-  val dutyStatement = AccountType("DutyDefermentStatement")
-  val c79cert = AccountType("C79Certificate")
-  val sercStatement = AccountType("SecurityStatement")
-  val PostPonedVATStatement = AccountType("PostponedVATStatement")
+  val dutyStatement: String = "Requested duty deferment statements"
+  val c79cert: String = "Requested import VAT certificates (C79)"
+  val sercStatement: String = "Requested notification of adjustment statements"
```

Comment:
What serc stands for ? Trying to make sure that it is aligned to covey the meaning for the value